### ISSUE TITLE
Create app-like effects

### ISSUE THREAD

**Issue by Djelibeybi** [delfick/photons#14]
It would be super peachy keen if Interactor could run app-like effects (like Pastel, Animate, Flicker, etc) across multiple devices that don't have their own internal firmware effects.

**Comment by Djelibeybi** [comment]
Implemented via #32

**Comment by delfick** [comment]
tile animations are part of it, but they only apply to tiles and don't do
things like flicker

**Comment by Djelibeybi** [comment]
Fair point. :) Issue reopened.

**Comment by Kackerlacka** [comment]
Hi, just wanted to comment on this thread to see if there has been any activity on adding Flicker to Photons. It's about Halloween time and I was hoping to use my bulbs to flicker some orange colors using Photons. No worries if not, just figured I'd revive the thread!

**Comment by delfick** [comment]
nah, I've barely done anything on photons for two years. Only flicker thing
remains the script I mention at the top of this issue, which does work
reasonably well.

**Comment by delfick** [comment]
seems that link doesn't work, it lives here now
https://github.com/delfick/photons/blob/main/examples/flicker
